dashboard/src/components/Header/Header.v2.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  angleIcon,
+  applicationsIcon,
+  ClarityIcons,
+  clusterIcon,
+  fileGroupIcon,
+} from "@clr/core/icon-shapes";
+import * as React from "react";
+import { NavLink } from "react-router-dom";
+import { CdsIcon } from "../Clarity/clarity";
+
+import logo from "../../logo.svg";
+import { IClusterState } from "../../reducers/cluster";
+import { app } from "../../shared/url";
+import "./Header.v2.css";
+
+ClarityIcons.addIcons(applicationsIcon, clusterIcon, fileGroupIcon, angleIcon);
+
+interface IHeaderProps {
+  authenticated: boolean;
+  fetchNamespaces: () => void;
+  logout: () => void;
+  cluster: IClusterState;
+  defaultNamespace: string;
+  push: (path: string) => void;
+  setNamespace: (ns: string) => void;
+  createNamespace: (ns: string) => Promise<boolean>;
+  getNamespace: (ns: string) => void;
+}
+
+class Header extends React.Component<IHeaderProps> {
```

Comment:
Since this is just a render function, why not a function component here?

Reply:
I was just waiting for you to merge your PR, I can do that now in a separate PR.